packages/react/src/components/IconButton/IconButton.stories.js

```diff
@@ -57,6 +57,10 @@ Default.args = {
 
 Default.argTypes = {
   align: {
+    // TODO:
+    // 1. Should the deprecated options be deleted?
+    // 2. The list doesn't include all of the options available in the
+    //    component. Is it supposed to?
```

Comment:
Yes, the deprecated options should be deleted and use the new options instead. I also checked with @thyhmdo on caret alignment and confirmed that [toolip placements](https://carbondesignsystem.com/components/tooltip/usage/#placement) should not include `-start`/`-end` for `left`/`right` and they should default to the center alignment.

Those alignments should be used for [popover](https://carbondesignsystem.com/components/popover/usage/#caret-tip).